app/assets/javascripts/oxalis/api/api_latest.js

```diff
@@ -729,6 +729,7 @@ class DataApi {
     const dataset = Store.getState().dataset;
 
     return doWithToken(token => {
+      // todo: use tracingStore maybe?
```

Comment:
Is this todo still valid? The method description makes it look like the method can be used to download segmentation data as well? Would the request for the segmentation data need to go to the tracing- or the datastore? Or both? ^^ :/